### ISSUE TITLE
Use user profile picture instead of world thumbnail for join/leave notifications

### ISSUE THREAD

**Issue by amplified1** [Yellow-Dog-Man/Resonite-Issues#1963]
### Is your feature request related to a problem? Please describe.

Profile pictures are highly recognizable and can be used to quickly identify someone you are familiar with without reading the text contents of the notification, and the world thumbnail just doesn't feel like the best use of that space in terms of aesthetics and utility in my opinion.

### Describe the solution you'd like

replace, or have an option to replace (more customization is always good) the world thumbnail shown in join/leave notifications with the joining/leaving user's profile picture. Much like user online notifications.

### Describe alternatives you've considered

both could be shown, but that may be too busy, another setting possibility perhaps.

### Requesters

@\_amplified\_

**Comment by epicEaston197** [comment]
You're experiencing a bug I see profile pictures

**Comment by Frooxius** [comment]
Yes, it should show both the profile and session thumbnail.

Can you provide a screenshot of what your notifications look like?

**Comment by epicEaston197** [comment]
> Yes, it should show both the profile and session thumbnail.

Wait I've only ever seen profile pictures. How is the session thumbnail supposed to fit both?. And what would the use case be showing the session thumbnail?.

**Comment by shiftyscales** [comment]
Session thumbnail could give additional context about the specific world, e.g. if two sessions were just called "User's World".

I just saw the profile pic and session thumbnail in a notification, so as far as I can see, it is working as intended.

Does the user you received the notification about have a profile image assigned, @amplified1?

**Comment by amplified1** [comment]
didn't realize it was a bug, I will do some testing (disabling mods, etc) and give information when I can

**Comment by epicEaston197** [comment]
> Session thumbnail could give additional context about the specific world, e.g. if two sessions were just called "User's World".

I don't see this being useful because right now user join leave notifications is scoped to the current section you're in I have yet to see a profile picture and a section thumbnail that would only make sense if the join leave notifications notified you of users leaving in sessions you're not currently focused in

**Comment by shiftyscales** [comment]
Well, that was requested in #1943, @epicEaston197 - but I don't see any harm in it being there regardless.

**Comment by epicEaston197** [comment]
> Well, that was requested in #1943, @epicEaston197 - but I don't see any harm in it being there regardless.

Yeah that's understandable that's gonna be a setting in the future but I don't understand how you're receiving session thumbnails that is what's confusing me here

**Comment by shiftyscales** [comment]
It is on the right side of the notification- do you not see them, @epicEaston197?

**Comment by epicEaston197** [comment]
> It is on the right side of the notification- do you not see them, @epicEaston197?

No I don't believe I've ever seen a session thumbnail on the right side of the user join/leave notification it's either that or I completely ignored it or the thumbnail just didn't load in

**Comment by epicEaston197** [comment]
I can confirm I do not see a session thumbnail image only a profile picture @shiftyscales 
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/453bc684-893b-41b4-a638-5c24a887934d)

**Comment by shiftyscales** [comment]
Hmm- is that a session you are hosting, @epicEaston197? Or someone else's? What is the visibility of that session?

**Comment by epicEaston197** [comment]
> Hmm- is that a session you are hosting, @epicEaston197? Or someone else's? What is the visibility of that session?

It was the session I wasn't hosting and it was a publicly viewable session

**Comment by epicEaston197** [comment]
okay now i'm only getting session thumbnails 
this user had a profile icon it's either I get one or the other
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/2d0a02d9-a3d3-46dd-b4d2-a155775a9f63)

**Comment by epicEaston197** [comment]
okay now I'm receiving both
(I've not closed my game since my last comment)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/6ea5b595-8cdb-4ec6-99e1-cd3df0f838b5)
